CRM/Event/Form/ParticipantFeeSelection.php

```diff
@@ -390,4 +407,23 @@ private function emailReceipt(array $params): void {
     CRM_Core_BAO_MessageTemplate::sendTemplate($sendTemplateParams);
   }
 
+  /**
```

Comment:
Seems like $this->_eventId would always be defined, so do we need to have a method to get it here?

Reply:
Or if there is a reason to have it, I think we can just have it return $this->_eventId

Reply:
SO where I am going with these is the idea that you should be able to call `getEventID()` & similar `get`functions at any point & they don't rely on some order of events where things are `set` first, but you would always get ... the event ID.

Also - I've marked as supported for external use - with the goal of providing some consistency to extensions wanting to access basic information about what is being processed (without directly accessing the properties which could be inconsistent)